manifests/base/deployment.yaml

```diff
@@ -14,17 +13,23 @@ spec:
     metadata:
       labels:
         control-plane: kubeflow-training-operator
+      annotations:
+        sidecar.istio.io/inject: "false"
     spec:
-      #      securityContext:
-      #        runAsNonRoot: true
       containers:
         - command:
             - /manager
-          # disable leader-elect now
-          #        args:
-          #        - --leader-elect
           image: kubeflow/training-operator:v1.0.0
```

Comment:
The last suggestion from my side.
I think we can keep `image: kubeflow/training-operator:v1.0.0` here.
We don't have `v1.0.0` tag for now and we can just keep the latest version of image here.
What do you think @Jeffwan ?

Reply:
Good point. Since the tag doesn't exist. It's better to remove it and don't confuse users. I remove the tag